### ISSUE TITLE
docExpansion issue

### ISSUE THREAD

**Issue by ocdevops** [swagger-api/swagger-ui#3197]
- swagger-ui version
3.0.13
It think it does not work, it is ignoring docExpansion parameter settings.

**Comment by wollanup** [comment]
+1
I can't get `operationsSorter` and `docExpansion` to work.
Is this syntax correct ?
in [index.html](https://github.com/swagger-api/swagger-ui/blob/master/dist/index.html) : 

```
const ui = SwaggerUIBundle({
		operationsSorter  : "alpha",
		docExpansion      : 'none',
		url               : http://localhost/doc",
		dom_id            : '#swagger-ui',
		presets           : [
			SwaggerUIBundle.presets.apis,
			SwaggerUIStandalonePreset
		],
		plugins           : [
			SwaggerUIBundle.plugins.DownloadUrl
		],
		layout            : "StandaloneLayout"
	})
```

**Comment by siddamreddy** [comment]
Hello wollanup,

How are u getting SwaggerUIBundle object. I couldn't get.
import { SwaggerUIBundle, SwaggerUIStandalonePreset } from 'swagger-ui-dist'.
I am using angular2
const ui = SwaggerUIBundle({
		apisSorter : "alpha",
		docExpansion : "none",
		url : "http://localhost/doc",
		dom_id : '#swagger-ui',
		presets : [
			SwaggerUIBundle.presets.apis,
			SwaggerUIStandalonePreset
		],
		plugins : [
			SwaggerUIBundle.plugins.DownloadUrl
		],
		layout : "StandaloneLayout"
	})

**Comment by webron** [comment]
`docExpansion` is available in `master`, not 3.0.13. It will be available in 3.0.14.
